spec/factories/posts.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :post do
+    user_id 1
+content "MyString"
+  end
+
+end
```

Comment:
When I use the second solution, FactoryGirl tries to create the user several times, which fails email uniqueness validation. What should I do to avoid that?